word_guess_owl.rb

```diff
@@ -0,0 +1,153 @@
+# Class to print out owl ASCII art for the game.
+# Could potentially be used to change difficulty level of game by limiting initial number of guesses
+class Owl
+  def initialize(initial_zz)
+    @initial_zz = initial_zz
+  end
+
+  attr_reader :initial_zz
+
+  def sleepy_owl
+    puts " {~;~}"
+    puts " /)__)"
+    puts "/  ;;"
+  end
+
+  def zzzz(wrong_guesses)
+    string = "\n     "
+    (@initial_zz - wrong_guesses).times do
+      string += "Z "
+    end
+    puts string
+  end
+
+  def awake_owl
+    puts " {O;O}"
+    puts " /)__)"
+    puts "/  ;;"
+  end
+
+  def winning
+    puts "\n     HOOOOO-RAY!"
+    puts " {^;^}"
+    puts " /)__)"
+    puts "/  ;;"
+  end
+end
+# Allows user to guess letters, validates their input and prompts again if not valid,
+# stores letters already guessed, and increments the number of wrong guesses,
+# generates blank letter spaces, and fills in the blanks as game progresses.
+class Game
+  def initialize (word)
+    @finished_word = word
+    @letters = word.upcase.chars
+    @wrong_guesses = 0
+    @guessed_letters = []
+    @word_array =@letters.length.times.collect {|x| "_"}
+  end
+
+  attr_reader :letters, :finished_word, :guessed_letters, :wrong_guesses, :word_array
```

Comment:
Coding Style Note:  As a rule, put your `attr_reader` at the top of the class above even `initialize`.